Dockerfile

```diff
@@ -1,34 +1,33 @@
-FROM node:12-stretch
-LABEL description="Debian image to build nativefier apps"
+FROM node:12-alpine
+LABEL description="Alpine image to build nativefier apps"
 
-# Get wine32, not 64, to work around binary incompatibility with rcedit.
-# https://github.com/nativefier/nativefier/issues/375#issuecomment-304247033
-# Forced us to use Debian rather than Alpine, which doesn't do multiarch.
-RUN dpkg --add-architecture i386
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get --yes install wine32 imagemagick \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add bash wine imagemagick dos2unix \
+    && rm -rf /var/lib/apk/lists/*
+
+WORKDIR /nativefier
```

Comment:
The `electron-packager` uses a package called `rcedit` to generate its packages. It appears that was updated to call wine64 instead of wine if its host os was 64 bit, so with us having wine32 hard coded in there, it was failing do to an inability to find wine64.